src/stub.js

```diff
@@ -1,6 +1,6 @@
 export const UserStub = {
   id: 1,
-  username: "Dick",
```

Comment:
How is this a cleanup? This should be replaced with real data as part of the authentication.

Reply:
I think that this is being handled in @amyd11 's PR, https://github.com/CMSgov/blueprint_ui/pull/62

Reply:
Exactly @jwhite3 this entire file will be replaced with real data at that time.